static/index.html

```diff
@@ -177,6 +175,20 @@ <h3>Last error</h3>
           />
         </div>
 
+        <div class="btn-group mb-5">
+          <button @click="saveForm" class="btn btn-primary">Save</button>
+
+          <button
+            @click="resetForm"
+            class="btn btn-outline-primary"
+            id="reset-config"
+          >
+            Reset
+          </button>
+        </div>
+
```

Comment:
In the playground, I moved the save and reset buttons above the switches, as it's actually only the text fields that need to be saved (the switches auto-save when they're changed).

Reply:
Haha, TIL. Never looked at the code but always hit Save after changing the switches 🙈